spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -1979,6 +1979,12 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
     module.quantize()
   }
 
+  def createDLEstimator() = {
+  }
+
+  def createDLClassifier() = {
+  }
```

Comment:
Are the above two methods used?

Reply:
removed, originally used to test the javaValue.jvm_class_constructor